apache/zeppelin
# ZEPPELIN-964 ] Jobmanagement - (2) basic front-end

### What is this PR for?
Job manger basic front end.
You can check to paragraph and notebook information.
It is created with the following additional functions PR.
(E. G., Filter and sort)

this PR is divided from the https://github.com/apache/zeppelin/pull/921

### What type of PR is it?
Featrue

### Todos
- [x] - notebook information
- [x] - viewing status for paragraph
- [x] - feat. running progress bar.
- [x] - added job menu in navbar. 
(It can be modified by the following (https://github.com/apache/zeppelin/pull/1013)


### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-964

### How should this be tested?
1. create and running, modified notebook and paragraph.
2. check to cron or normal notebook status in job manager.

### Screenshots 

<img width="683" alt="job manger-basic" src="https://cloud.githubusercontent.com/assets/10525473/16113612/0120dec8-33f8-11e6-8dec-c74048fae637.png">

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? yes
